src/main/java/core/basesyntax/strategy/filestrategy/DataParserHandler.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy.filestrategy;
+
+import core.basesyntax.strategy.filestrategy.impl.CsvDataParserImpl;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+public class DataParserHandler {
+    private final Map<String, DataParser> dataParsers = new HashMap<>();
```

Comment:
```suggestion
    private final Map<String, DataParser> dataParsersMap;
```
initialize using constructor